src/org/openlcb/InitializationCompleteSimpleMessage.java

```diff
@@ -0,0 +1,28 @@
+package org.openlcb;
+
+// For annotations
+import net.jcip.annotations.*; 
+import edu.umd.cs.findbugs.annotations.*; 
+
+/**
+ * Initialization Complete message implementation
+ * with the SImple Protocol modifier set
+ *
+ * @author  Bob Jacobsen   Copyright 2024
+ * @version $Revision$
+ */
+@Immutable
+@ThreadSafe
+public class InitializationCompleteSimpleMessage extends InitializationCompleteMessage {
+    
+    public InitializationCompleteSimpleMessage(NodeID source) {
```

Comment:
if we do it with a new class, we should add a function `isSimpleProtocolSubset()` that is false in the base class and true here.

Alternatively, we could avoid making new classes and instead take the "isSimple" bit on the constructor. The producer/consumer identified messages do that; when one m,essage has variants that are different in some minor ways.
I don't have a strong preference.

Reply:
The next PR will do this with a property on the messages and `hasSimpleProtocol()`